### PR TITLE
batches: take two at explaining rollout window behaviour

### PR DESCRIPTION
**Pull request by LawnGnome**

### CODE REVIEW

File: doc/admin/config/batch_changes.md [hunk 1]
```diff
@@ -10,10 +10,20 @@ Configuring rollout windows allows changesets to be created and updated at a slo
 
 Rollout windows are configured through the `batchChanges.rolloutWindows` [site configuration option](site_config.md). If specified, this option contains an array of rollout window objects that are used to schedule changesets. The format of these objects [is given below](#rollout-window-object).
 
+### Behavior
```
Comment (eseliger) [inline doc/admin/config/batch_changes.md:13]: Did you put this single word in a caption, just to make clear that you didn't like my comment from last time? 😆 🇺🇸

File: doc/admin/config/batch_changes.md [hunk 2]
```diff
@@ -10,10 +10,20 @@ Configuring rollout windows allows changesets to be created and updated at a slo
...
+
 When rollout windows are enabled, changesets will initially enter a **Scheduled** state when their batch change is applied. Hovering or tapping on the changeset's state icon will provide an estimate of when the changeset will be reconciled.
 
 To restore the default behavior, you can either delete the `batchChanges.rolloutWindows` option, or set it to `null`.
 
+Or, to put it another way:
+
+| `batchChanges.rolloutWindows` configuration | Behaviour |
```
Comment (eseliger) [inline doc/admin/config/batch_changes.md:21]:
```suggestion
| `batchChanges.rolloutWindows` configuration | Behavior |
```

zorry :)

**Review by eseliger — APPROVED** [review]

File: doc/admin/config/batch_changes.md (hunk 2 above)
Comment (LawnGnome) [reply doc/admin/config/batch_changes.md:21]: DAMMIT

(I promise I wasn't trolling you.)

**Review by malomarrec — APPROVED** [review]
Awesome!

**Pull request merged**